### PR TITLE
LPD-7822

### PR DESCRIPTION
**Pull request by BryanEngler**

### CODE REVIEW

**Comment by BryanEngler** [comment]
ci:test:sf

**Comment by BryanEngler** [comment]
ci:test:relevant

**Comment by BryanEngler** [comment]
ci:test:search

File: modules/apps/portal-search/portal-search-spi/src/main/java/com/liferay/portal/search/spi/index/configuration/contributor/CompanyIndexConfigurationContributor.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 /**
  * @author Adam Brandizzi
  */
-public interface IndexConfigurationContributor {
```
Comment (gustavolimav) [inline modules/apps/portal-search/portal-search-spi/src/main/java/com/liferay/portal/search/spi/index/configuration/contributor/CompanyIndexConfigurationContributor.java]: good rename

File: modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/settings/SettingsBuilder.java [hunk 2]
```diff
@@ -15,7 +16,7 @@
 /**
  * @author André de Oliveira
  */
-public class SettingsBuilder {
```
Comment (gustavolimav) [inline modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/settings/SettingsBuilder.java]: Thats the way I did too, it just weird to have builder implementing helper

File: modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-api/src/main/java/com/liferay/portal/search/elasticsearch7/configuration/ElasticsearchConfiguration.java [hunk 3]
```diff
@@ -108,7 +108,7 @@ public interface ElasticsearchConfiguration {
 	public String indexNumberOfReplicas();
 
 	@Meta.AD(
-		deflt = "", description = "number-of-index-shards-help",
+		deflt = "1", description = "number-of-index-shards-help",
```
Comment (gustavolimav) [inline modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-api/src/main/java/com/liferay/portal/search/elasticsearch7/configuration/ElasticsearchConfiguration.java:111]: That was a great idea, really liked that way without use the default file.

**Comment by BryanEngler** [comment]
ci:test:search

File: modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/settings/SettingsBuilder.java (hunk 2 above)
Comment (BryanEngler) [reply modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/settings/SettingsBuilder.java]: ya, i just renamed SettingsBuilder to SettingsHelperImpl

**Comment by BryanEngler** [comment]
ci:report:2567261292

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by BryanEngler** [comment]
ci:report:2567636334

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by BryanEngler** [comment]
ci:report:2568040627

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:report:2568154142

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by BryanEngler** [comment]
ci:test:sf

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by BryanEngler** [comment]
ci:report:2568249680

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by BryanEngler** [comment]
ci:report:2568276896

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by BryanEngler** [comment]
ci:report:2568367205

**Pull request closed without merging**